channels-rke2.yaml

```diff
@@ -1,14 +1,14 @@
 releases:
-  - version: v1.18.19+rke2r1
+  - version: v1.18.20+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.19.11+rke2r1
+  - version: v1.19.12+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.7+rke2r2
+  - version: v1.20.8+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.1-rc3+rke2r1
+  - version: v1.21.2-rc1+rke2r1
```

Comment:
LGTM after update to remove rc and redo go generate

Reply:
When it's ready for review just include Kinara as a 2nd reviewer.

Reply:
Sounds good Dave. Thanks. 

Tagging @kinarashah